app/src/main/java/com/github/mik629/android/fundamentals/ui/global/ActorItemAdapter.kt

```diff
@@ -8,43 +8,47 @@ import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.github.mik629.android.fundamentals.BuildConfig
 import com.github.mik629.android.fundamentals.GlideRequest
+import com.github.mik629.android.fundamentals.R
 import com.github.mik629.android.fundamentals.databinding.ActorItemBinding
-import com.github.mik629.android.fundamentals.domain.model.ActorItem
+import com.github.mik629.android.fundamentals.domain.model.Actor
 
 class ActorItemAdapter(
     private val glideRequest: GlideRequest<Drawable>
-) : ListAdapter<ActorItem, ActorItemAdapter.ViewHolder>(DIFF_CALLBACK) {
+) : ListAdapter<Actor, ActorItemAdapter.ViewHolder>(ActorItemAdapterDiffCallback()) {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder =
-        ViewHolder(ActorItemBinding.inflate(LayoutInflater.from(parent.context), parent, false))
+        ViewHolder(
+            ActorItemBinding.inflate(LayoutInflater.from(parent.context), parent, false),
+            glideRequest
+        )
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         holder.updateViewItem(getItem(position))
     }
 
-    inner class ViewHolder(private val binding: ActorItemBinding) :
+    class ViewHolder(
+        private val binding: ActorItemBinding,
+        private val glideRequest: GlideRequest<Drawable>
+    ) :
         RecyclerView.ViewHolder(binding.root) {
 
-        fun updateViewItem(item: ActorItem) {
-            with(binding) {
-                name.text = item.name
-                if (!item.ava.isNullOrEmpty()) {
-                    glideRequest.centerCrop()
-                        .load("${BuildConfig.BASE_IMAGE_URL}${item.ava}")
-                        .into(ava)
-                }
+        fun updateViewItem(item: Actor) {
+            binding.name.text = item.name
+            if (!item.photoUrl.isNullOrEmpty()) {
+                glideRequest.centerCrop()
+                    .load("${BuildConfig.BASE_IMAGE_URL}${item.photoUrl}")
```

Comment:
{BuildConfig.BASE_IMAGE_URL} already should be inside in photoUrl
should be added on mapping
and as I remember must come from config but not form BuildConfig